ankiweb.go

```diff
@@ -51,14 +46,30 @@ func (a *AnkiAccount) Load(filename string) error {
 		make([]*http.Cookie, 0),
 		"",
 	}
-	err = json.Unmarshal(data, &storedData)
+
+	fileExists := true
+
+	data, err := ioutil.ReadFile(filename)
```

Comment:
I guess `switch-case` will reflect the logic better, than additional variable `fileExists`:

```go
data, err := ioutil.ReadFile(filename)

switch {
    case os.IsNotExist(err):
        shouldAuth = true

    case err == nil:
        err = json.Unmarshal(data, &storedData)
        if err != nil {
            return false, err
        }

    default:
        return false, err
}
```